Vignettes/usingcchsflow.Rmd

```diff
@@ -108,12 +111,12 @@ Use the `bind_rows` function to combine datasets and create a new dataset name.
 
 ### Example 2. Transform a single variable from multiple CCHS datasets
 
-This example shows how you can transform and combine a variable across multiple CCHS cycles. The sex variable in CCHS 2001 (`DHHA_SEX`) and CCHS 2013 (`DHH_SEX`) is transformed a common variable (`DHH_SEX`) and combined into a single dataset. In `cchsflow` the CCHS 2007 variable name is used as the common variable name. See the [variableDetails vignette]() for more information.
+This example shows how you can transform and combine a variable across multiple CCHS cycles. The sex variable in CCHS 2001 (`DHHA_SEX`) and CCHS 2013 (`DHH_SEX`) is transformed a common variable (`DHH_SEX`) and combined into a single dataset. In `cchsflow` the CCHS 2007 variable name is used as the common variable name. See the [variableDetails vignette](articles/variableDetails.html) for more information.
```

Comment:
No need for 'articles' see https://github.com/Big-Life-Lab/bllflow/blob/3a91aa80b3d7ad83f137eb86fdbc16b956dc7550/vignettes/bllflow.Rmd#L47 
for example